vehicle/tesla/controller.go

```diff
@@ -15,19 +15,21 @@ const ProxyBaseUrl = "https://tesla.evcc.io"
 
 type Controller struct {
 	vehicle *tesla.Vehicle
+	ble     bool
 	current int64
 	dataG   provider.Cacheable[float64]
 }
 
 // NewController creates a vehicle current and charge controller
-func NewController(ro, rw *tesla.Vehicle) *Controller {
+func NewController(ro, rw *tesla.Vehicle, ble bool) *Controller {
 	v := &Controller{
 		vehicle: rw,
+		ble:     ble,
 	}
 
 	v.dataG = provider.ResettableCached(func() (float64, error) {
-		if v.current >= 6 {
-			// assume match above 6A to save API requests
+		if v.current >= 6 || v.ble {
+			// assume match above 6A to save API requests OR if bleVehicle is active
 			return float64(v.current), nil
```

Comment:
I would be cautious with that as, IMHO, it will bring back issue #13007 where going below 5A is ignored by the Tesla the first time. This feedback loop allows evcc to detect the current mismatch and send the command again.

Reply:
We don't have this issue with ble. I've used it for multiples days now. And I always charge with less then 5A.

Reply:
Ok, good to read.

I’m also using your ble proxy with TeslaMate and I did face this issue however. Maybe I’m a special case as I also charge a non-Tesla vehicle on my TWC3 ;-)

Anyway I fixed it by fetching the actual current from TeslaMate. I added that in my custom vehicle definition and then evcc started to detect and automatically fix the current mismatch:
```
GetMaxCurrent:
    source: mqtt
    topic: teslamate/cars/2/charger_actual_current
    timeout: 8h
```

Reply:
That's annoying. However, if we query the Tesla server every time there is a change in charging speed, we will hit the API limit. Therefore, I think this query must be removed if we want to avoid the API limits.